tests/backends/test_tpu_backend.py

```diff
@@ -45,9 +46,8 @@ def test_resume_training_on_cpu(tmpdir):
         max_epochs=1,
         default_root_dir=tmpdir,
     )
-    result = trainer.fit(model)
-
-    assert result == 1
+    trainer.fit(model)
+    assert trainer.state == TrainerState.FINISHED, "Training failed with %s" % trainer.state
```

Comment:
Feel like updating them all again into f-strings (more mordern)? :)

```suggestion
    assert trainer.state == TrainerState.FINISHED, f"Training failed with {trainer.state}"
```

Reply:
Agree !

Reply:
the diff is lazy and always eval but as you wish...